wallet/wallet.c

```diff
@@ -484,6 +484,10 @@ struct utxo **wallet_utxo_boost(const tal_t *ctx,
 		if (utxo_is_reserved(utxo, blockheight))
 			continue;
 
+		/* Don't add csv-locked ones */
+		if (utxo_is_csv_locked(utxo, blockheight))
+			continue;
```

Comment:
Could this be a fix for: https://github.com/ElementsProject/lightning/issues/6973?

Reply:
Yes, definitely!